starter_code/components.js

```diff
@@ -0,0 +1,41 @@
+class Picture extends React.component {
+  constructor(){
+    super();
+    this.state = { image: 'http://images.fanpop.com/images/image_uploads/wooderson---Slater-dazed-and-confused-301811_400_300.jpg'}
+  }
+  _revealMyImage(image) {
```

Comment:
Inside this method, I would then toggle that state to the opposite of whatever it is. 

```js
this.setState({image: !this.state.image});
```